api/translate/translate/v1/translate.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package translate.v1;
+
+import "google/api/annotations.proto";
+
+message Message {
+  string id = 1;
+  string translation = 2;
+  bool fuzzy = 3;
+}
+
+message Messages {
+  string language = 1;
+  repeated Message messages = 2;
+  map<string, string> labels = 3;
+}
+
+message UploadTranslationFileRequest {
+  string language = 1;
+  bytes data = 2;
+}
+
+message UploadTranslationFileResponse {}
```

Comment:
Empty response?

Reply:
@klavinsjanis , @VladislavsPerkanuks will do 2 PRs for this ticket to unblock @DarjaLeonova sooner.